.changelog/d02faa8f0bdd4ac88dea1a6f95c98de5.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "d02faa8f-0bdd-4ac8-8dea-1a6f95c98de5",
+    "type": "bugfix",
+    "description": "revert commit of validating PutObject content length causing integration test failure",
```

Comment:
Just remove this changelog. It basically never happened since it didn't release

Reply:
The changelog is needed since catapult dry run test checks the changelog for the commit change